integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/e2e/openstack/OpenStackTests.java

```diff
@@ -0,0 +1,222 @@
+
+    @Inject
+    private StackTestClient stackTestClient;
+
+    @Inject
+    private ClusterDefinitionTestClient clusterDefinitionTestClient;
+
+    @Test(dataProvider = "openStackHdpTestsDataProvider")
+    public void testOpenStackWithDefaulkHDPBlueprintsAndPrewarmedImagesWithoutGateWaySetup(
+            TestContext testContext,
+            String clusterDefinitionName,
+            String[] groups,
+            String scaleGroup,
+            @Description TestCaseDescription description) {
+
+        testContext
+                .given(ClusterTestDto.class)
+                .withClusterDefinitionName(clusterDefinitionName)
+                .given(StackTestDto.class)
+                .withInstanceGroupsEntity(getInstanceGroupTestDtos(testContext, groups))
+                .withCluster(ClusterTestDto.class.getSimpleName())
+                .when(stackTestClient.createV4())
+                .await(STACK_AVAILABLE)
+                .when(stackTestClient.scalePostV4()
+                        .withGroup(scaleGroup)
+                        .withDesiredCount(2))
+                .await(STACK_AVAILABLE)
+                .when(stackTestClient.scalePostV4()
+                        .withGroup(scaleGroup)
+                        .withDesiredCount(1))
+                .await(STACK_AVAILABLE)
+                .when(stackTestClient.stopV4())
+                .await(STACK_STOPPED)
+                .when(stackTestClient.startV4())
+                .await(STACK_AVAILABLE)
+                .then((tc, testDto, cc) -> stackTestClient.deleteV4().action(tc, testDto, cc))
+                .validate();
+    }
+
+    @DataProvider(name = "openStackHdpTestsDataProvider")
+    public Object[][] openStackHdpTestsDataProvider() {
+        SparklessTestContext testContext = getBean(SparklessTestContext.class);
+        minimalSetupForClusterCreation(testContext);
+        ClusterDefinitionTestDto select = testContext
+                .given(ClusterDefinitionTestDto.class)
+                .when(clusterDefinitionTestClient.listV4());
+        List<ClusterDefinitionV4ViewResponse> viewResponses = select.getViewResponses()
+                .stream()
+                .filter(e -> e.getStatus().equals(ResourceStatus.DEFAULT))
+                .filter(e -> isHdp(e) && notASharedServiceClusterDefinition(e))
+                .collect(Collectors.toList());
+
+        Object[][] data = new Object[viewResponses.size()][5];
+        for (int i = 0; i < viewResponses.size(); i++) {
+            data[i][0] = testContext;
+            data[i][1] = viewResponses.get(i).getName();
+            data[i][2] = getHdpGroups();
+            data[i][3] = getHdpScaleGroup();
+            data[i][4] = new TestCaseDescription.TestCaseDescriptionBuilder()
+                    .given("there is an available cloudbreak")
+                    .when("a stack create request with "
+                            + viewResponses.get(i).getName() + " AND scale AND stop AND start requests are sent")
```

Comment:
I think you should change this: `viewResponses.get(i).getName() + " AND scale AND stop AND start requests are sent"` to `viewResponses.get(i).getName() + " cluster definition is sent AND scale AND stop AND start requests are sent"`.

So please add the term "cluster definition is sent". The whole text will then look like this:
`When a stack create request with EDW-ETL: Apache Hive, Apache Spark 2 stack definition is sent AND scale AND stop AND start requests are sent`
This makes sense and it's grammatically correct.